UnicodeJsps/src/main/java/org/unicode/jsp/UnicodeUtilities.java

```diff
@@ -1734,29 +1738,36 @@ class PropertyAssignment {
             boolean isNew = assignment.first == Settings.LATEST_VERSION_INFO;
             String versionRange =
                     (showVersion ? (isSingleVersion ? first : first + ".." + last) + ": " : "");
-            if (assignment.value == null) {
-                out.append("<td" + defaultClass + ">" + versionRange + "<i>null</i></td>");
-            } else {
-                String hValue = toHTML.transliterate(assignment.value);
-                out.append(
-                        "<td"
-                                + defaultClass
-                                + ">"
-                                + (isMultivalued
-                                        ? ""
-                                        : ("<a target='u' "
-                                                + (isNew ? "class='changed' " : "")
-                                                + "href='list-unicodeset.jsp?a=[:"
-                                                + (isCurrent ? "" : "U" + last + ":")
-                                                + propName
-                                                + "="
-                                                + hValue
-                                                + ":]'>"))
-                                + versionRange
-                                + hValue
-                                + (isMultivalued ? "" : "</a>")
-                                + "</td>");
+            var htmlValue = new StringBuilder();
+            for (int i = 0; i < assignment.values.size(); ++i) {
```

Comment:
very optional: with some fiddling, this should be able to become a stream where we map null vs. other, then use a string joiner...

Reply:
I briefly thought of that, but I am not sure it would be more readable. I have a tendency to write Rust in that kind of extremely functional style, maybe I should try it in Java too…

Will toy with it and report back.

Reply:
The result seems surprisingly palatable.